src/java/picard/illumina/IlluminaBasecallsToSam.java

```diff
@@ -95,11 +95,35 @@
  * @author mccowan@broadinstitute.org
  */
 @CommandLineProgramProperties(
-        usage = IlluminaBasecallsToSam.USAGE,
-        usageShort = IlluminaBasecallsToSam.USAGE,
+        usage = IlluminaBasecallsToSam.USAGE_SUMMARY + IlluminaBasecallsToSam.USAGE_DETAILS,
+        usageShort = IlluminaBasecallsToSam.USAGE_SUMMARY,
         programGroup = Illumina.class
 )
 public class IlluminaBasecallsToSam extends CommandLineProgram {
+    static final String USAGE_SUMMARY = "Transforms a lane of Illumina data file formats (*.bcl, *.locs, *.clocs, *.qseqs, etc.) into the" +
+            "SAM or BAM file format."  ;
+    static final String USAGE_DETAILS = "<p>The IlluminaBasecallsToSam tool enables the transformation of Illumina binary call (*.bcl) data to unmapped SAM/BAM files.  " +
+            "The unmapped BAM file is often referred to as a uBAM. </p>" +
+            "" +
+            "<p>For each lane of a flowcell, the BARCODES_DIR file is produced by the <a href='http://broadinstitute.github.io/picard/command-line-overview.html#ExtractIlluminaBarcodes'>ExtractIlluminaBarcodes</a> tool.  " +
```

Comment:
This first sentence should come later and should be rephrased to state that the BARCODES_DIR file can be obtained with blah. Right now it sounds like the most important thing you want to say about Tool X is that Tool Y does something somewhat relevant.